test/DynamoCoreTests/Configuration/PreferenceSettingsTests.cs

```diff
@@ -199,5 +203,136 @@ public void TestSerializationTrustedLocations()
 
             Assert.IsTrue(settingsLoaded.IsTrustedLocation(Path.GetTempPath()));
         }
+
+        private List<string> PropertiesWithStaticValues()
+        {
+            List<string> staticProperties = new List<string>() { "PythonTemplateFilePath","DefaultPythonEngine","NodeSearchTagSizeLimit","IronPythonResolveTargetVersion"};            
```

Comment:
If really required to be hardcoded, why not just make this a static member of the class?

Reply:
refactored.